<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>S2F</class>
 <widget class="QMainWindow" name="S2F">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>280</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>530</width>
    <height>280</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Spm to friction</string>
  </property>
  <property name="iconSize">
   <size>
    <width>128</width>
    <height>128</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>580</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>580</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>580</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Расчет по одному полю</string>
     </attribute>
     <widget class="QLineEdit" name="lineEditIn_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BrowseInButton_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>30</y>
        <width>75</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Обзор</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Открыть файл</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Spm2FrictionButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Расчет трения из spm1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="About3Button">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>О функции</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ExitButton_9">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>60</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Расчет по набору полей</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Открыть папку</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Spm2FrictionButton_Dir">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Расчет трения из spm1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ExitButton_11">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>50</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BrowseInButton_5">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>75</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Обзор</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditIn_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="About4Button">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>О функции</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>381</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Расчет по одному полю(txt)</string>
     </attribute>
     <widget class="QLineEdit" name="lineEditBw">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BrowseBwButton">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>60</y>
        <width>75</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Обзор</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Открыть файл обратного хода</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Открыть файл прямого хода</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditFw">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BrowseFwButton">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>75</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Обзор</string>
      </property>
     </widget>
     <widget class="QPushButton" name="calcFButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Расчитать трение из txt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="About2Button">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>90</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>О функции</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ExitButton_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>90</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>spm1 в txt</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Открыть файл</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditIn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BrowseInButton">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>30</y>
        <width>75</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Обзор</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Spm2TxtButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Перевод spm1 в txt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ExitButton">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>60</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
     <widget class="QPushButton" name="About1Button">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>О функции</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Константы для расчетов</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>213</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="3">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Коэффициент
жесткости,Н/м</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Высота иглы, м</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Коэффициент
Пуассона</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="koeffEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="sEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="vEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>170</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" rowspan="3">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Нагрузка, %/100</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Калибровачный
коэффициент</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Длина 
кантеливера, м</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="procentEdit">
       <property name="inputMask">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="koeffkalEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>541</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>ExitButton_2</sender>
   <signal>clicked()</signal>
   <receiver>S2F</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ExitButton_9</sender>
   <signal>clicked()</signal>
   <receiver>S2F</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ExitButton</sender>
   <signal>clicked()</signal>
   <receiver>S2F</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ExitButton_11</sender>
   <signal>clicked()</signal>
   <receiver>S2F</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
